packages/babel-types/src/definitions/core.ts

```diff
@@ -1960,19 +1961,36 @@ defineType("TaggedTemplateExpression", {
   },
 });
 
+function templateElementCookedValidator(node: t.TemplateElement) {
+  const raw = node.value.raw;
+  if (node.value.cooked != null) {
+    return;
+  }
+
+  let cooked = null;
+  try {
+    cooked = unraw(raw);
+  } catch (error) {}
+  node.value.cooked = cooked;
+}
+templateElementCookedValidator.type = "{ raw: string; cooked?: string }"; // hack
```

Comment:
Can you leave a comment that the hack is used to generate AST typings?

Reply:
I tried to modify this but it didn't work, so I modified another place.